puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
I am not sure what's going on, if you look at the output of http://ci.theforeman.org/job/test_katello_core/database=postgresql,ruby=1.9.3,slave=fast/16683/console it appears to have the correct gemset enabled and in use, but the bundle install doesn't indicate that it is.